jekyll/asset/style/main.scss

```diff
@@ -85,6 +85,7 @@ body {
 header, .main, section, footer {
   @extend .clearfix;
   width: 100%;
+  padding: 0 .5rem;
```

Comment:
Everything was touching the left side of the screen when it was narrow & it really bothered me.